reactor-core/src/main/java/reactor/core/publisher/SignalType.java

```diff
@@ -51,9 +51,17 @@
 	 */
 	ON_COMPLETE,
 	/**
-	 * A signal that can be propagated AFTER an operator has terminated
+	 * A signal when an operator completes
+	 */
+	AFTER_TERMINATE,
+	/**
+	 * A context read signal that is pulled
+	 */
+	CURRENT_CONTEXT,
+	/**
+	 * A context signal that can be propagated from parent source
```

Comment:
rephrase to "A signal that a context was propagated from parent to source"? (or is it from source to parent?)